### ISSUE TITLE
samples/samples/empty.wav

### ISSUE THREAD

**Issue by anytizer** [LMMS/lmms#6880]
It is a list of TripleOscillator presets that refer to non-existing empty.wav file.
Escalated from #6630 and old version of LMMS that caused a rare crash.

![image](https://github.com/LMMS/lmms/assets/5563341/c1510ccb-f410-4e0e-ac4f-b0cc08748c8e)

Also, the final file path has repeated word "samples".
May be a text edit to refer correct empty.wav should be fine in those .xpf

**Comment by michaelgregorius** [comment]
Hi @anytizer! I have just found your issue after I have created #6896. I have now added your information there as well.

Is it ok, if I close this issue? My report also has some steps to reproduce, etc.

**Comment by anytizer** [comment]
Yes @michaelgregorius, you can close this. Only, Additional information is about the duplicate word being present: "samples/samples/" to access empty.wav.

**Comment by michaelgregorius** [comment]
Ok, thanks!

Closing this issue as there is #6896 with some additional information.
